layout: fix npm run build:watch

# Bug

### Expected behavior

npm run build:watch works

### Actual behavior

Fails with: 

```
~/dev/design-system/packages/layout(feat/layoutPageWidth ✔) nrbw

> @pluralsight/ps-design-system-layout@5.0.6 build:watch /Users/jaketrent/dev/design-system/packages/layout
> npm run build -- --watch


> @pluralsight/ps-design-system-layout@5.0.6 build /Users/jaketrent/dev/design-system/packages/layout
> run-s build:js build:cjs build:css "--watch"

ERROR: Invalid Option: --watch
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @pluralsight/ps-design-system-layout@5.0.6 build: `run-s build:js build:cjs build:css "--watch"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @pluralsight/ps-design-system-layout@5.0.6 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jaketrent/.npm/_logs/2019-12-16T21_54_17_234Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @pluralsight/ps-design-system-layout@5.0.6 build:watch: `npm run build -- --watch`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @pluralsight/ps-design-system-layout@5.0.6 build:watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jaketrent/.npm/_logs/2019-12-16T21_54_17_250Z-debug.log
```
### Steps to repro

1. cd packages/layout
2. npm run build:watch

--- a/packages/layout/package.json
+++ b/packages/layout/package.json
@@ -13,7 +13,7 @@
     "build:cjs": "babel --root-mode upward src --out-dir dist/cjs",
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
     "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm",
-    "build:watch": "npm run build -- --watch",
+    "build:watch": "npm run build:js -- --watch",
     "prepublish": "npm run build",
     "storybook": "start-storybook -p 6006",
     "test": "jest",

--- a/packages/theme/package.json
+++ b/packages/theme/package.json
@@ -12,7 +12,7 @@
     "build": "run-s build:js build:cjs",
     "build:cjs": "babel --root-mode upward src --out-dir dist/cjs",
     "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm",
-    "build:watch": "npm run build -- --watch",
+    "build:watch": "npm run build:js -- --watch",
     "prepublish": "npm run build",
     "test": "jest",
     "test:watch": "npm run test -- --watchAll"

